ironfish/src/primitives/evmDescription.ts

```diff
@@ -11,3 +13,30 @@ export interface EvmDescription {
   r: Buffer
   s: Buffer
 }
+
+export function legacyTransactionToEvmDescription(tx: LegacyTransaction): EvmDescription {
+  return {
+    nonce: BigInt(tx.nonce),
+    to: tx.to ? Buffer.from(tx.to.bytes) : Buffer.alloc(0),
+    value: BigInt(tx.value),
+    data: Buffer.from(tx.data),
+    v: Number(tx.v),
+    r: tx.r ? Buffer.from(bigIntToBytes(tx.r)) : Buffer.alloc(0),
+    s: tx.s ? Buffer.from(bigIntToBytes(tx.s)) : Buffer.alloc(0),
+  }
+}
+
+export function evmDescriptionToLegacyTransaction(desc: EvmDescription): LegacyTransaction {
+  return new LegacyTransaction({
+    nonce: desc.nonce,
+    to: desc.to.length > 0 ? desc.to : undefined,
+    value: desc.value,
+    data: desc.data,
+    v: BigInt(desc.v),
+    r: desc.r.length > 0 ? bytesToBigInt(desc.r) : undefined,
+    s: desc.s.length > 0 ? bytesToBigInt(desc.s) : undefined,
+    // TODO(jwp) gas constants are hardcoded
```

Comment:
I have this hardcoded, we could potentially just add to the description for now, wdyt?

Reply:
Is there a way to generically disable gas int eh EVM?  If so we should do it.